### ISSUE TITLE
cant zone

### ISSUE THREAD

**Issue by erfg12** [erfg12/EQTrilogy-Old#11]
Need to find out why.

**Comment by erfg12** [comment]
Fixed by replacing zone_line_nodes in query to zone_points

**Comment by erfg12** [comment]
But X and Y are backwards and some zone point names are incorrect or missing.
EDIT: Fixed by using eqmac's zone lines. BUT, some are STILL WRONG!?

**Comment by cursian** [comment]
I have been doing a lot of work on this and correcting the database. I noticed there is no database included here?

**Comment by erfg12** [comment]
Any files you need (database, client, etc.) you can find at https://newagesoldier.com/everquest-classic-source-code-eqclassic-org/

**Comment by erfg12** [comment]
Cursian do you have the zone line fixes?

**Comment by erfg12** [comment]
Found a fix for zone issues.

**1)** The database had x,y,z at 0,0,0 in lots of zones. These have to be set to their location.
**2)** The range I set to 75 by default. Little did I know, 75 is waaaay too big. Needs to be about 5, otherwise the zone line is going to be really far off.
**3)** Since I used the Mac EQEmu database for zone lines, the Z axis is really far off. For example, felwithea was -108. It's supposed to be -10. Not sure why it's that far off, but it is. So that needs to be fixed as well.

When building these new zones in the DB, use http://newagesoldier.com/myfiles/eqtrainer/telescripts/
